docs/adr/0007-minimal-model.md

```diff
@@ -0,0 +1,46 @@
+# 7. Minimal Model
+
+Date: 2019-01-21
+
+## Context
+
+We currently have several classes that we think of as models.
+
+   * ActiveRecord classes that reflect persisted data in the database. These are conventionally found in `app/models`.
+   * Plain Ruby classes that reflect readonly data stored in configuration files. These currently have no place to live :-(.
+
+We also have other classes that ressemble models, but arent't e.g.
```

Comment:
"ressemble" => "resemble"
"arent't" => "aren't"

Reply:
Thanks for spotting - fixed :-).